apps/cms/src/modules/ui/instructions.tsx

```diff
@@ -32,6 +32,18 @@ export const Instructions = () => {
               tab to add an instance of your provider. Click <q>Add provider</q>, and select the cms
               provider you want to use. Fill in the configuration form and hit <q>Save</q>.
             </li>
+            <li>
+              Go to your CMS website and prepare product variant model shape with:
```

Comment:
To consider in the future - probably you can create such data model with the API. Testers and users will be thankful ;)